### DISCUSSION TITLE
/hyndsight/seasonal-mortality-rates/

### DISCUSSION THREAD

**Discussion by robjhyndman** [robjhyndman/robjhyndman.com#320] [Announcements]
Seasonal mortality rates | Rob J Hyndman 
 https://robjhyndman.com/hyndsight/seasonal-mortality-rates/

**Comment by robjhyndman** [comment]
```header
Originally posted by George on 2020-05-24 15:24:08
````
Re England &amp; Wales, the spike at the start of 2015 and 2018 is due to
higher than usual influenza activity.\
See
[https://assets.publishing.s\...](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/839350/Surveillance_of_influenza_and_other_respiratory_viruses_in_the_UK_2018_to_2019-FINAL.pdf &quot;https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/839350/Surveillance_of_influenza_and_other_respiratory_viruses_in_the_UK_2018_to_2019-FINAL.pdf&quot;)

**Comment by joanagomesdias** [comment]
Thanks, Rob, for the very insightful post. 
Unfortunately, the link to the file is not working. Here is the new link: "https://www.mortality.org/File/GetDocument/Public/STMF/Outputs/stmf.csv"

**Reply by robjhyndman** [reply]
Thanks. I've now updated the link
